tiatoolbox/models/engine/patch_predictor.py

```diff
@@ -44,6 +44,90 @@ def __init__(
 class PatchPredictor:
     """Patch-level predictor.
 
+    The models provided by tiatoolbox should give the following results:
+
+    .. list-table:: PatchPredictor performance on the Kather100K dataset [1]
+       :widths: 15 15
+       :header-rows: 1
+
+       * - Model name
+         - F_1 score
```

Comment:
Can we add this as F1 or with "1" as a subscript?

Reply:
https://docutils.sourceforge.io/docs/ref/rst/roles.html#subscript

Reply:
Resolved in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/335/commits/018a504fc133ef9bfa5c2753e0fb3aacfb681cb9